.github/workflows/docs.yml

```
@@ -0,0 +1,29 @@
+name: Docs
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  docs:
+    name: Publish docs
+    runs-on: ubuntu-latest
+
```

```suggestion
    environment:
      name: github-pages
      url: ${{ steps.deployment.outputs.page_url }}

```
This information is required, see https://github.com/actions/starter-workflows/blob/main/pages/static.yml